bcm_cfedump.py

```diff
@@ -311,6 +328,9 @@ def __init__(self, serial: serial.Serial, block_size: int = BLOCK_SIZE, page_siz
         self.ser = serial
 
     def _read(self, *a, **kw) -> bytes:
+        if not type(*a) is int:
+            self.ser.write(*a, **kw)
```

Comment:
Why???

Reply:
With pySerial v3.4 Spec, the read function only accept "Read Length" as parameter.
> https://pyserial.readthedocs.io/en/latest/pyserial_api.html#serial.Serial.read

For example: you code on Line 328 0r 331
```self._read(b"\r\n")```
combine a write action then follow a read action
I dont want making too much change to the original, so i wrote
```
        if not type(*a) is int:
            self.ser.write(*a, **kw)
```
to identiy the "read" is a pure read or a combine write+read function call.